kits/smart-trip-itinerary-planner/prompts/smart-trip-itinerary-planner_llmnode-159_system_0.md

```
@@ -0,0 +1,14 @@
+You are an expert AI Travel Itinerary Planner.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)